src/GlobalPayments.Api/Terminals/TerminalUtilities.cs

```diff
@@ -61,6 +62,35 @@ public class TerminalUtilities {
             return new DeviceMessage(buffer.ToArray());
         }
 
+        public static DeviceMessage BuildRequest(string message, ConnectionModes settings) {

```

Comment:
Pretty sure this method is superfluous.

Reply:
The reason why we overload the BuildRequest method is that Ingenico's different types of connection mode has also different frame formats. For Serial the data is prefixed with STX and suffixed with ETX and LRC while for IP connection the data is prefixed with HEADER.